docs/user-guide/ze-profiles.md

```diff
@@ -21,28 +30,34 @@ Create a team configuration file.
 4. Select **Create a New Team Configuration File**.
 5. If no workspace is open, a global configuration file is created. If a workspace is open, chose either a global configuration file or a project-level configuration file.
 6. Edit the config file to include the host information and save the file.
-7. Refresh Zowe Explorer by either clicking the button in the notification message shown after creation, `alt+z`, or the `Zowe Explorer: Refresh Zowe Explorer` command palette option.
+7. Refresh Zowe Explorer by either clicking the button in the notification message shown after creation, `alt` + `z`, or the `Zowe Explorer: Refresh Zowe Explorer` command palette option.
 
-Your team configuration file appears either in your `.zowe` folder if you choose the global configuration file option, or in your workspace directory if you choose the project-level configuration file option. The notification message that shows in VS Code after config file creation will include the path of the file created.
+    Your team configuration file appears either in your `.zowe` folder if you choose the global configuration file option, or in your workspace directory if you choose the project-level configuration file option. The notification message that displays in VS Code after the configuration file creation includes the path of the file created.
 
 ### Managing profiles
 
-You can edit your project-level or global configuration files.
-
-**Follow these steps**:
+Change profile validations and edit the profiles in your project or global configuration files.
 
 1. Right-click on your profile.
-2. Select the **Add**, **Update**, or **Delete Profile** options to edit the zowe config file in place.
+2. Select the **Manage Profile** option to choose from several authentication and profile management actions for the credentials detected in your Zowe Explorer session.
```

Comment:
is "session" correct here?

Reply:
Although a somewhat overloaded term, I think Session is correct here. 😋